indigo/analysis/terms/base.py

```diff
@@ -176,10 +174,10 @@ def find_term_references(self, doc, terms):
             return
 
         # term to term id
-        term_lookup = {v: k for k, v in terms.iteritems()}
+        term_lookup = {v: k for k, v in terms.items()}
 
         # big regex of all the terms, longest first
-        terms = sorted(terms.itervalues(), key=lambda t: -len(t))
+        terms = sorted(iter(terms.values()), key=lambda t: -len(t))
```

Comment:
I don't think `iter` is necessary.

```suggestion
        terms = sorted(terms.values(), key=lambda t: -len(t))
```